packages/scripts/src/upgrade-tests/voting2/1_Propose.ts

```diff
@@ -86,10 +95,37 @@ async function main() {
   });
   console.log("3.c. Transfer ownership of existing voting to voting upgrader:", transferExistingVotingOwnershipTx.data);
 
+  // Transfer Ownable contracts to VotingUpgraderV2
+  for (const ownableToMigrate of Object.entries(ownableContractsToMigrate).slice(ownableContractsToMigrate.length)) {
```

Comment:
When reading a struct from a contract it returns an object of type `[] & {}`
This operation keeps only the object part:
`ownableContractsToMigrate.slice(ownableContractsToMigrate.length)`

Maybe for clarity I could unroll this for loop and the following.